web/themes/custom/votegov/src/sass/components/date.scss

```
@@ -7,3 +7,7 @@
     @include u-margin-top(4);
   }
 }
+
+.vote-election-date {
```

thoughts on making this match a bit more to the one above in naming?

```suggestion
.vote-date--election {
```